### PR TITLE
Update region lines parsing to round to +/- MAX_VALUE

### LINKED ISSUE #467: [IR] parsing: region lines
The [region lines](https://github.com/web-platform-tests/wpt/blob/master/webvtt/parsing/file-parsing/tests/regions-lines.html) text parsing test is failing.
Some of the assertions pass but some fail due to the usage of very large number values. Safari is clamping the value to the max integer value while Firefox returns a 0. The spec just says to interpret the value as an integer but provides no guidance on it. This potentially means there's no way to test these edge cases. Though, may not be necessary.
Firefox's strict parser and Safari's non-standard implementations prevent the test from passing.

### PR DESCRIPTION
**Pull request by gkatsev**
This would potentially require an update to the WPT.

The text is borrowed from https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#rules-for-parsing-floating-point-number-values, though, I don't think that it's a great fit here.

I haven't regenerated the index.html yet, will do so after I figure out the process around bikeshed updates.

Fixes #467.

***
<a href="https://pr-preview.s3.amazonaws.com/gkatsev/webvtt/pull/470.html" title="Last updated on May 30, 2026, 4:26 PM UTC (4f11bc5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webvtt/470/f6b3eab...gkatsev:4f11bc5.html" title="Last updated on May 30, 2026, 4:26 PM UTC (4f11bc5)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -2852,7 +2852,19 @@ Objects</a>.</p>
 
        <li><p>Interpret |value| as an integer, and let |number| be that number.</p></li>
 
-       <li><p>Let |region|'s <a>WebVTT region lines</a> be |number|.</p></li>
+       <li><p><i>Conversion</i>: Let <var>S</var> be the set of finite IEEE 754 double-precision
```
Comment (gkatsev) [inline index.bs]: It's likely we don't need this `conversion` label

Comment (silviapfeiffer) [reply index.bs]: It may be useful if we need to do this conversion elsewhere as well. Might be worth checking.

**Comment by silviapfeiffer** [comment]
I would recommend adding this to the REC branch directly.

Then, for the master branch, update Bikeshed in a separate PR before making another PR with updates from the REC.

**Review by silviapfeiffer — APPROVED** [review]

**Comment by gkatsev** [comment]
Yeah, I can create another bikeshed PR and then rebase this. I'll add it to #460 as well.
Should I fixup the bikeshed update as outlined in this commit? https://github.com/w3c/webvtt/commit/754f13e3cf03d6036c3e4628c6920d17b412f778 or is it possible it may not be necessary?
